guides/keras_cv/fine_tune_via_textual_inversion.py

```diff
@@ -0,0 +1,691 @@
+"""
+Title: Teach StableDiffusion New Concepts via TextualInversion
+Authors: Ian Stenbit, [lukewood](https://lukewood.xyz)
+Date created: 2022/09/25
+Last modified: 2022/09/25
+Description: Learning new concepts in KerasCV's StableDiffusion Implementation
+"""
+
+"""
+# Textual Inversion
+
+StableDiffusion: the first high quality open source image generator.  Since its
+release, StableDiffusion has quickly become a favorite amongst the machine learning
+commun1ity.  The high volume of traffic has led to open source contributed improvements,
+heavy prompt engineering, and even the invention of novel algorithms.
+
+Perhaps the most impressive new algorithm being used is
+[Textual-inversion](https://github.com/rinongal/textual_inversion), presented in
+[_An Image is Worth One Word: Personalizing Text-to-Image Generation using Textual
+Inversion_](https://textual-inversion.github.io/)
+
+Textual-inversion is the process of teaching an Image-Generator the concept of a specific
+thing through the use of FineTuning. In the diagam below, you can see an example of
+this process where the authors teach the model new concepts, calling them "S_*".
+
+![https://i.imgur.com/KqEeBsM.jpg](https://i.imgur.com/KqEeBsM.jpg)
+
+Conceptually, textual inversion works by learning a token embedding for a new text
+token, keeping the remaining components of StableDiffusion frozen.
+
+This guide shows you how to fine-tune the StableDiffusion model shipped in KerasCV
+using the Textual-Inversion algorithm.
+
+First, lets start by important the packages we will need, and constructing
+StableDiffusion so we can use some of the sub-components for fine-tuning.
+"""
+
+"""shell
+!pip install -q git+https://github.com/keras-team/keras-cv.git
+!pip install -q luketils
+!pip install -q tensorflow==2.11.0
+"""
+import keras_cv
+from keras_cv import layers as cv_layers
+import tensorflow as tf
+from tensorflow import keras
+from luketils import visualization
```

Comment:
should keras tutorial have personal repo import?

Reply:
I personally don't see an issue with using it on keras.io as we use lots of third party modules.  

If we want to prioritize a visualization sub-package thats fine by me, but I figure by shipping it outside of Keras we can drop backwards compatibility concerns and prevent the need to copy-paste code across all of the tutorials that plot image galleries.

WDYT?

Reply:
Honestly, I just got sick of copy-pasting this and a few other utilities all over the place so I figured I may as well package them up and save myself the time.  Open to alternatives of course

Reply:
Sounds good, removed